source/fuzz/fuzzer_context.cpp

```diff
@@ -153,6 +153,7 @@ const std::pair<uint32_t, uint32_t> kChanceOfReplacingParametersWithStruct = {
 const std::pair<uint32_t, uint32_t> kChanceOfSplittingBlock = {40, 95};
 const std::pair<uint32_t, uint32_t> kChanceOfSwappingConditionalBranchOperands =
     {10, 70};
+const std::pair<uint32_t, uint32_t> kChanceOfSwappingFunctions = {10, 10};
```

Comment:
This gives a lower bound and an upper bound on the probability with which functions will be swapped. 10% seems like an OK lower bound, but I suggest a higher upper bound. Let's go with 90%. So {10, 10} should become {10, 90}.

Reply:
Changed to 90 upperbound.